tests/run_tests.sh

```diff
@@ -58,12 +94,21 @@ for PYTHON in /opt/python/*/bin/python; do
 		echo "invalid repaired wheel name"
 		exit 1
 	fi
-	${PYTHON} -m pip install ${REPAIRED_WHEEL}
-	if [ "$(${PYTHON} -c 'import forty_two; print(forty_two.answer())')" != "42" ]; then
-		echo "invalid answer, expecting 42"
-		exit 1
+	if [ "${IMPLEMENTATION}" != "graalpy" ]; then
+		${PYTHON} -m pip install ${REPAIRED_WHEEL}
+		if [ "$(${PYTHON} -c 'import forty_two; print(forty_two.answer())')" != "42" ]; then
+			echo "invalid answer, expecting 42"
+			exit 1
+		fi
```

Comment:
@timfel, the wheel just built can't be installed by pip, there's a mismatch in the platform tag: `linux_aarch64` vs `linux_armv7l`.

Reply:
```
[root@95647b9a4112 /]# graalpy3.10 -m pip debug
...
Compatible tags: 27
  graalpy310-graalpy230_310_native_aarch64_linux-linux_armv7l
  ...
  [First 10 tags shown. Pass --verbose to show all.]

[root@95647b9a4112 /]# graalpy3.10 -c "from packaging.tags import sys_tags; print(next(sys_tags()))"
graalpy310-graalpy230_310_native-linux_armv7l
```
This time packaging agrees but also gets it wrong. I guess this is related to the `is_32bit` check.

Reply:
Yes, the `is_32bit` check is biting us. The PR you linked didn't move forward mostly because of concern for 32-bit macOS. We have found making sys.maxsize larger on GraalPy brings other issues though, since some code assumes that (as documented) this gives the maximum array size.